### DISCUSSION TITLE
这个报错到底想让我干嘛

### DISCUSSION THREAD

**Discussion by VansamaZWT** [liusy58/NJU_AdvancedProgramming_SP22#83] [Q&A] [answered]
如图
![~~J_ZQ 6O7VKR)WRG88YBCP](https://user-images.githubusercontent.com/100075314/156004547-653ec757-8d48-436f-9310-9b61f4a55a2e.png)
![DUZKQ~@$DOC_1)1T RS7VVY](https://user-images.githubusercontent.com/100075314/156004608-e8e28604-bb73-4f2b-9b25-ed244300f7e0.png)

**Comment by bliss-bug** [comment]
是不是少了using namespace std;

**Comment by RicoloveFeng** [comment, marked as answer]
冷静一下，仔细看看代码，只发两张图别人很可能无法帮你解决任何问题，如果还是找不出问题可以私聊助教。

**Comment by VansamaZWT** [comment]
感谢，问题已经私聊解决
